images-list

```diff
@@ -888,6 +890,8 @@ neuvector/manager rancher/mirrored-neuvector-manager 5.2.4
 neuvector/manager rancher/mirrored-neuvector-manager 5.2.4-s1
 neuvector/manager rancher/mirrored-neuvector-manager 5.3.0
 neuvector/manager rancher/mirrored-neuvector-manager 5.3.2
+neuvector/manager rancher/mirrored-neuvector-manager 5.3.3
+neuvector/prometheus-exporter rancher/mirrored-neuvector-prometheus-exporter 1-1.0.0
```

Comment:
what's the deal with this one?

Reply:
@brandond we have changed the tag format to 1-1.0.0 and it is not going to be changed very often as other images. is it okay?

Reply:
Yeah was just wondering if this is a one-off or if the versioning was changed.

Reply:
versioning has been changed. Can you also please merge. Thanks.